assets/js/data/payment/test/check-payment-methods.tsx

```diff
@@ -133,6 +134,9 @@ const registerMockPaymentMethods = ( savedCards = true ) => {
 	wpDataFunctions
 		.dispatch( PAYMENT_STORE_KEY )
 		.__internalUpdateAvailablePaymentMethods();
+	wpDataFunctions.dispatch( CART_STORE_KEY ).receiveCart( {
+		payment_methods: [ 'cheque', 'bacs', 'credit-card' ],
+	} );
```

Comment:
We have a TS error in this line. We need a `CartResponse` object. We can use the `previewCart` object which include already these 3 payment methods:

```js
import { previewCart } from '@woocommerce/resource-previews';

wpDataFunctions.dispatch( CART_STORE_KEY ).receiveCart( previewCart );
```